### PR TITLE
add partner-data namespace

### PR DESCRIPTION
**Pull request by williamhgough**
- Adds `partner-data` namespace
- fixes namespace deployment row formatting.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -19,12 +19,13 @@ mod pages;
...
 use pages::{deployment::DeploymentPage, index::IndexPage, namespace::NamespacePage};
 
-const NAMESPACES: [&str; 5] = [
+const NAMESPACES: [&str; 6] = [
     "partner",
     "partner-platform-api",
     "partner-commission",
     "partner-auth-internal",
     "partner-auth-external",
+    "partner-data",
```
Comment (johanbrandhorst) [inline src/lib.rs]: We should add an issue to make this dynamic from the `ListDeployments`

Comment (williamhgough) [reply src/lib.rs]: Agreed, can't remain static, I was thinking more of adding an endpoint for `ListNamespaces`, can run a simple SQL query to return unique namespaces in the deployments table?

Comment (johanbrandhorst) [reply src/lib.rs]: Absolutely yes

Comment (williamhgough) [reply src/lib.rs]: Have added an issue, will pick up on Friday - https://github.com/utilitywarehouse/partner-version-app/issues/5

**Pull request merged**